R/ContinuousDataFunctions.R

```diff
@@ -0,0 +1,671 @@
+#' TADA_listNWIS
+#'
+#' List available USGS daily NWIS data
+#'
+#' @description
+#' Retrieves available metadata from USGS National Water Information System (NWIS) based on
+#' different spatial queries: area of interest (AOI), specific sites, or state boundaries.
+#' Returns a spatial sf object containing site information and available parameters.
+#' If no data is found, returns an empty sf object with appropriate column structure.
+#'
+#' @param aoi_sf An sf object defining the area of interest. All individual sf
+#' features (or "rows" in the sf data frame) must be under 118,078 square miles
+#' (roughly the area of Nevada).
+#' @param statecode Character vector of two-letter state codes (e.g., c("CA", "OR")).
+#' @param sites Character vector of USGS site numbers.
```

Comment:
suggest to change from sites to siteid for consistency with other TADA data retrieval functions. I am working on this now